.github/workflows/CI_Labels.yml

```diff
@@ -0,0 +1,64 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI Labels
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    types:
+      # default types (i.e. default for on: pull_request)
+      - opened
+      - synchronize
+      - reopened
+      # trigger on change of labels
+      - labeled
+      - unlabeled
+    branches:
+      - develop-v3
+      - develop-v2
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Labels
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const VALID_LABELS_MESSAGE = 'Success! PR has valid labels.'
+            const BREAKING_LABELS = ['Breaking Change', 'Non-breaking Change']
+            const CHANGE_TYPE_LABELS = ['feature', 'bug', 'docs', 'enhancement']
```

Comment:
I think we have `Schema: Documentation` instead of `docs` now? I'm not sure when this changed but my latest PR didn't pass the check because of it..

Reply:
Cool cool, I'll fix that, thanks Jie 👍